setup.py

```diff
@@ -57,6 +57,7 @@ def package_files(directory):
         'netifaces==0.10.6',
         'ipaddress==1.0.19',
         'PyYAML==3.10',
+        'psutil==3.3.0',
```

Comment:
Given that we're only targeting one OS, and we are only using this to retrieve a single value, might it be worth us getting this information another way (e.g. /proc/meminfo)?

Reply:
We are already using `psutil` to find out about memory in cloudify-manager (https://github.com/cloudify-cosmo/cloudify-manager/blob/master/rest-service/manager_rest/storage/storage_manager.py#L492).  I also usually like the idea of _standing on the shoulders of giants_ (in this case meaning using a well-established library to do the _dirty work_ for me).  

How much are you leaning towards grepping `/proc/meminfo`?

Reply:
Not especially strongly- it'll add an extra dependency to this, but given that it's not the first one I'm not overly bothered about it.

Interesting note: os.sysconf('SC_PAGE_SIZE') * os.sysconf('SC_PHYS_PAGES')
(Just found out about that one).
Anyway, I'll mark this as resolved.